packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -105,6 +115,9 @@ export class ExecutionEngineHttp implements IExecutionEngine {
   // It's safer to to avoid false positives and assume that the EL is syncing until we receive the first payload
   private state: ExecutionEngineState = ExecutionEngineState.ONLINE;
 
+  // Cache EL client version from the latest getClientVersion call
+  private executionClientVersion: ClientVersion[] = [];
```

Comment:
Do we really care about this being an array? I don't see any use case right now from Lodestar pov, it's not used in the graffit either.

Might wanna go with something like this

```suggestion
  private clientVersion?: ClientVersion;
```